config/base.webpack.plugins.js

```diff
@@ -23,23 +23,23 @@ plugins.push(WriteFileWebpackPlugin);
  * @type {var}
  */
 const HtmlWebpackPlugin = new (require('html-webpack-plugin'))({
-    title: 'My App',
+    title: 'Home',
```

Comment:
I think that it should be `Red Hat Cloud services` am I right?

Reply:
We're not `Red Hat Cloud Services`. I've asked mrike about it, so this needs BU approval.

Reply:
@karelhala mrike says to use "Home"